packages/flutter/lib/src/rendering/sliver_fill.dart

```diff
@@ -113,7 +113,17 @@ class RenderSliverFillRemaining extends RenderSliverSingleBoxAdapter {
   /// the remaining space in the viewport.
   RenderSliverFillRemaining({
     RenderBox child,
-  }) : super(child: child);
+    @required this.scrollBody,
+  }) : assert(scrollBody != null),
+       super(child: child);
+
+  /// Whether the child has a scrollable body
+  ///
+  /// Defaults to false such that the child will fill the remainder of the
+  /// viewport, and not extend further.
+  /// Setting this value to true will allow for the child to extend beyond the
+  /// viewport and scroll, as seen in [NestedScrollView].
+  final bool scrollBody;
```

Comment:
This probably shouldn't be final. If the widget changes, we will need to update the render object.